README.rst

```diff
@@ -48,7 +48,7 @@ Links
     :target: https://readthedocs.org/projects/pydocstyle/?badge=latest
     :alt: Documentation Status
 
-* `Read the full documentation here <https://pydocstyle.org>`_.
+* `Read the full documentation here <http://pydocstyle.readthedocs.io>`_.
```

Comment:
I think the `https` part can be left unchanged.

https://pydocstyle.readthedocs.io